### PR TITLE
UT cloud credentials project assignment

### PR DESCRIPTION
**Pull request by eranelbaz**
### Issue & Steps to Reproduce / Feature Request
[//]: <> (choose “resolves” “fixes” or “closes” and put link for the issue)
UT for `resource_cloud_credentials_project_assignment.go`
### Solution
- Added UTs
- Fixed `*client.ApiClient`
- Set all client signs to have the same order of proj id and cred it

### CODE REVIEW

**Comment by eranelbaz** [comment]
[UT for env0/resource_cloud_credentials_project_assignment.go](https://trello.com/c/oJMgZb8g/3416-ut-for-env0-resourcecloudcredentialsprojectassignmentgo)

File: env0/resource_cloud_credentials_project_assignment_test.go [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+	"regexp"
+	"testing"
+)
+
+func TestUnitResourceCloudCredentialsProjectAssignmentResource_Create(t *testing.T) {
+	resourceType := "env0_cloud_credentials_project_assignment"
+	resourceName := "test"
+	accessor := resourceAccessor(resourceType, resourceName)
+	assignment := client.CloudCredentialsProjectAssignment{
+		CredentialId: "cred-it",
```
Comment (eyalatzmon) [inline env0/resource_cloud_credentials_project_assignment_test.go:16]: I think you can define 
`{		CredentialId: "cred-it",		ProjectId:    "proj-it",	}` and reuse it in few place in tests

Comment (eranelbaz) [reply env0/resource_cloud_credentials_project_assignment_test.go:16]: 62b2c69a took @avnerenv0 method for reuse stuff

**Review by eyalatzmon — APPROVED** [review]
Good work!!! :muscle:

**Pull request merged**